core/src/create.jl

```diff
@@ -225,13 +210,47 @@ end
 
 function FlowBoundary(db::DB, config::Config)::FlowBoundary
     static = load_structvector(db, config, FlowBoundaryStaticV1)
-    defaults = (; active = true)
-    static_parsed = parse_static(static, db, "FlowBoundary", defaults)
-    return FlowBoundary(
-        static_parsed.node_id,
-        static_parsed.active,
-        static_parsed.flow_rate,
-    )
+    time = load_structvector(db, config, FlowBoundaryTimeV1)
+
+    static = load_structvector(db, config, FlowBoundaryStaticV1)
+    time = load_structvector(db, config, FlowBoundaryTimeV1)
+
+    static_node_ids = Set(static.node_id)
+    time_node_ids = Set(time.node_id)
+    msg = "Flowboundary cannot be in both static and time tables"
+    @assert isdisjoint(static_node_ids, time_node_ids) msg
+    static_node_ids = collect(static_node_ids)
+    time_node_ids = collect(time_node_ids)
+    node_ids = get_ids(db, "FlowBoundary")
+
+    msg = "FlowBoundary node IDs don't match"
+    @assert issetequal(node_ids, union(static_node_ids, time_node_ids)) msg
+
+    control_mapping = Dict{Tuple{Int, String}, NamedTuple}()
```

Comment:
Where is this variable used?

Reply:
Nowehere :^)

Reply:
We really need some kind of static analyzer for that